roles/smartgateway/templates/deployment.yaml.j2

```diff
@@ -34,15 +35,15 @@ spec:
         - -tls-cert=/etc/tls/private/tls.crt
         - -tls-key=/etc/tls/private/tls.key
         - -cookie-secret-file=/etc/proxy/secrets/session_secret
-        - -openshift-service-account=NA
```

Comment:
I'll chalk this up to a change in the oauth container. It kinda felt weird to me that it worked without the extra RBAC to begin with.